project/src/main/java/rest/api/services/DreamTripsAPI.java

```diff
@@ -1,25 +1,41 @@
 package rest.api.services;
 
 import com.worldventures.dreamtrips.api.entity.model.EntityHolder;
+import com.worldventures.dreamtrips.api.hashtags.model.HashtagsSearchResponse;
 import com.worldventures.dreamtrips.api.profile.model.PrivateUserProfile;
 import com.worldventures.dreamtrips.api.trip.model.TripWithDetails;
 import com.worldventures.dreamtrips.api.trip.model.TripWithoutDetails;
+import okhttp3.MultipartBody;
+import okhttp3.ResponseBody;
 import retrofit2.Call;
-import retrofit2.http.GET;
-import retrofit2.http.Path;
-import retrofit2.http.Query;
+import retrofit2.http.*;
 
 import java.util.List;
 
 public interface DreamTripsAPI {
+
 	@GET("/api/profile")
 	Call<PrivateUserProfile> getCurrentUserProfile();
 
-	public static final String TRIPS_SEARCH_PATH = "/api/trips";
+	String TRIPS_SEARCH_PATH = "/api/trips";
```

Comment:
Yes, we'll use multiple requests with this path but different parameters sets, as soon as Hermet support for query parameters is implemented.